### PR TITLE
cicd: build push image on tags

### PR DESCRIPTION
**Pull request by sgaunet**
This PR add a job to build and push docker image on docker hub and also on github.

To push on docker hub, you will have to setup (settings/secrets and variables / actions) :

* an environment variable: DOCKERHUB_LOGIN
* a secret: DOCKERHUB_PASSWORD

I'm not very familiar with github actions. I'm waiting for your comments.

Best regards

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build/Push image ghcr.io/mcorbin/cabourotte:latest
```
Comment (mcorbin) [inline .github/workflows/publish.yml:20]: imo this part could be removed, I don't plan to support ghcr + docker hub for the image

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          fi
+      - name: Login to Docker Container Registry
+        uses: docker/login-action@v2
+        with:
+          username: ${{ vars.DOCKERHUB_LOGIN }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Tag and Push image to docker registry
+        shell: /usr/bin/bash {0}
+        run: |
+            docker tag ghcr.io/mcorbin/cabourotte:latest mcorbin/cabourotte:latest
```
Comment (mcorbin) [inline .github/workflows/publish.yml:40]: the repo is appclacks/cabourotte

File: .github/workflows/publish.yml (hunk 1 above)
Comment (sgaunet) [reply .github/workflows/publish.yml]: I've removed this part

File: .github/workflows/publish.yml (hunk 2 above)
Comment (sgaunet) [reply .github/workflows/publish.yml]: changed as demand

**Comment by sgaunet** [comment]
I've updated the PR in order to build/push on dockerhub, to appclacks/cabourotte registry. you will have to setup (settings/secrets and variables / actions) :

-     a secret: DOCKERHUB_LOGIN
-     a secret: DOCKERHUB_PASSWORD

**Comment by mcorbin** [comment]
Thank you

**Pull request merged**